src/MessagePack.GeneratorCore/Generator/ShouldUseFormatterResolverHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using MessagePackCompiler.CodeAnalysis;
+
+namespace MessagePackCompiler.Generator
+{
+    public static class ShouldUseFormatterResolverHelper
+    {
+        private static readonly string[] PrimitiveTypes =
+        {
+            "short",
+            "int",
+            "long",
+            "ushort",
+            "uint",
+            "ulong",
+            "float",
+            "double",
+            "bool",
+            "byte",
+            "sbyte",
+            "char",
```

Comment:
There's a similar list somewhere in the MessagePack library with the same purpose: should it use a resolver to get a formatter for it or treat it as a primitive. It would be nice to share the list if we can find it.

Reply:
I found the list and consolidated this. `byte[]` was missing on this list so I added it and fixed up the codegen to handle it properly.

Reply:
Thank you!